trade-finance-manager-ui/server/types/view-models/deal-cancellation/bank-request-date-view-model.ts

```diff
@@ -14,6 +14,7 @@ export type BankRequestDateValidationViewModel = {
 export type BankRequestDateViewModel = BaseViewModel & {
   ukefDealId: string;
   dealId: string;
+  previousPage: string;
```

Comment:
Not a blocker for this PR - seems like a lot of repetition in these view models - maybe `ukefDealId`, `dealId` and `previousPage` could go into `BaseViewModel` (i'm not sure what it's used for exactly - or maybe a new `DealViewModel`/`CaseViewModel`

Reply:
Have made a `BaseCancellationViewModel` for the 3 view models to build up from 👍